vanttec_uuv/Obstacle[] obstacles